src/game/node.rs

```diff
@@ -51,6 +51,7 @@ impl GameNode for PostFlopNode {
 
     #[inline]
     fn regrets(&self) -> &[f32] {
+        // TODO(Jacob): Something is causing either storage2 pointer to be off, or num_elements to be too big
```

Comment:
@JacobVanGeffen can we resolve this todo?

Reply:
Yes, can delete this.